### ISSUE TITLE
Checkbox Label read twice when 'position: relative'

### ISSUE THREAD

**Issue by chris-wassenaar-at-kiran** [nvaccess/nvda#8885]
If a checkbox is inside a label that has 'position: relative' on it, when you tab to the checkbox the label is read twice.

I have attached a very simple example illustrating the issue.

### Steps to reproduce:
- Download [test.zip](https://github.com/nvaccess/nvda/files/2520085/test.zip)
- Unzip html file
- Turn on NVDA
- open html file
- Tab through page to second checkbox

### Actual behavior:
- First checkbox is read once
- Second checkbox is read twice
![image](https://user-images.githubusercontent.com/17224407/47581257-85142d00-d916-11e8-86ad-c9bec9adc3d8.png)

### Expected behavior:
- Both checkboxes are read only once

### System configuration:
- NVDA + Chrome

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version: 
2018.3.2

#### Windows version:
Version	10.0.17134 Build 17134

#### Name and version of other software in use when reproducing the issue:
Google Chrome: 69.0.3497.100

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by DrSooom** [comment]
I couldn't reproduce this issue with Firefox 56.0.2 and 63.0 and IE 11 on Win7x64 and NVDA 2018.1, but with Chrome 70.0.3538.77 (portable version, 32-bit). To be exactly:

### Actual behaviour with Chrome 70.0.3538.77 and NVDA 2018.1:
+ When pressing TAB and Shift+TAB in focus mode, both checkboxes are spoken once.
+ When the cursor is located on the first checkbox, press TAB in browse mode and the second one will be spoken twice.
+ When the cursor is located on the second checkbox, press Shift+TAB in browse mode and the first one will be spoken twice now.
+ The first and the second checkbox are spoken once when navigating from the address bar back to the document with TAB or Shift+TAB while in browse and in focus mode.
+ Both checkboxes are spoken only once in browse mode when navigating with ArrowUp and ArrowDown.

**Comment by chris-wassenaar-at-kiran** [comment]
Thanks DrSooom!

- I retested "browse" vs "focus" mode and I agree that this only happens in "browse" mode.  
- However, I do not see that the first checkbox is ever spoken twice when I TAB / Shift+TAB back and forth.  Maybe the difference here has to do with our version mismatch.

Thank you for reviewing this!
- Chris

**Comment by Adriani90** [comment]
Duplicate of #9155.
